index.js

```diff
@@ -71,6 +184,46 @@ function compareValues(parent, theirs, mine, path) {
       theirs: theirs
     };
   }
+  else if(_.isArray(parent) || _.isArray(mine) || _.isArray(theirs)) {
+    // Maintain array order unless flagged as ignoreOrder
+    if (!_.isUndefined(options) && options['ignoreOrder']) {
+      parent = parent.sort();
+      mine = mine.sort();
+      theirs = theirs.sort();
```

Comment:
They all need to be arrays in order to call `sort` on all of them, right? It's only checking if one of them is.